docs/rfc/rfc-template.md

```diff
@@ -0,0 +1,35 @@
+# RFC {RFC-NUMBER}: {TITLE}
```

Comment:
```
---
sidebar_position: 1
---
```

?

Reply:
we dont have this on the adr section, ideally we tell it to not show the template files like we do today

Reply:
Yeah it is fine without it too. It was only required for the README to show properly I think.